docs/src/userguide/glossary.rst

```diff
@@ -0,0 +1,194 @@
+.. _glossary:
+
+Glossary
+=============
+
+.. glossary::
+
+   Cartopy
+        A python package for producing maps, and other geospatial data.
+        Allows plotting on these maps, over a range of projections.
+
+        | **Related:** :term:`Matplotlib`
+        | **More information:** `CartoPy Site <https://scitools.org.uk/cartopy/docs/latest/>`_
+        |
+
```

Comment:
I think I'd like us to reference CF conventions in a dedicated section, which I guess will go here.
I think we particularly need to be clearer about the relationship between CF and netcdf.
I suggest something like ...
```
   CF Conventions
        Rules for storing meteorological Climate and Forecast data in
        :term:`NetCDF Format` files, defining a standard metadata format to
        describe what the data is.
        This also forms the data model which iris is based on.
        See `CF Conventions <http://cfconventions.org/>`_.
```

Some of that slightly borrowed from the "NetCDF Format" entry 
-- which I'm also proposing changes to (below) ...